cmd/sql/flow_test.go

```diff
@@ -130,7 +130,7 @@ func TestFlowGenerateCmd(t *testing.T) {
 	err := execFlowCmd([]string{"init", projectDir}...)
 	assert.NoError(t, err)
 
-	err = execFlowCmd([]string{"generate", "example_basic_transform", "--project-dir", projectDir}...)
+	err = execFlowCmd([]string{"generate", "example_basic_transform", "--project-dir", projectDir, "--verbose"}...)
```

Comment:
Do we have tests for both when we enable `--verbose` and not? Do they have different assertions?

Reply:
Those tests are only testing that the inputs work with cobra. For example, if the flag is unknown, it will raise an error.

Reply:
We can add all flags at once to cover all scenarios of flags added to cobra.